packages/devtools_app/test/auto_dispose_mixin_test.dart

```diff
@@ -116,24 +116,6 @@ void main() {
       expect(values.length, equals(3));
       expect(values.last, equals(19));
     });
-
```

Comment:
this test is no longer relevant after https://github.com/flutter/flutter/commit/5f3bee55ad4804984256dc6bfd762fd0911db901